### DISCUSSION TITLE
Volunteering my services...

### DISCUSSION THREAD

**Discussion by thewhiterabbit** [coinfoundry/miningcore#1132] [General]
25+ years of software engineering experience here available for task assignment since I have a lot of free time for the next couple months.

Cool project btw.

**Comment by oliverw** [comment]
The best way to get started - as with any other open source project - is probably by submitting a PR for an open issue or feature request.

**Comment by thewhiterabbit** [comment]
Right! So, what would you like me to do first? Maybe others are working on
those already? Who is the team leader? When I approach any project it's
good to first attain a correct direction to travel in as not to waste time
working on something someone else already is...

I'm not gonna spend three days writing code to have a pull of the same
patch from someone else a few hours before mine is ready. Do you understand
my logic here? It wastes the communities par effort and that's bad.

Let me know who's working on what and I'll pick something that's not being
developed or has been dropped and needs picking up.

Happy Hump Day!

On Tue, Jan 25, 2022 at 10:54 AM Oliver Weichhold ***@***.***>
wrote:

> The best way to get started - as with any other open source project - is
> probably by submitting a PR for an open issue or feature request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/oliverw/miningcore/discussions/1132#discussioncomment-2045406>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIWJTW3TLGRVD4LI652QRDUX3WXJANCNFSM5MYWNLXQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Aaron Nall

*Ethan Vandal™*

**Reply by oliverw** [reply]
> Right! So, what would you like me to do first? Maybe others are working on those already? 

Others?

> Who is the team leader? 

Me, just without a team.

> When I approach any project it's good to first attain a correct direction to travel in as not to waste time working on something someone else already is... I'm not gonna spend three days writing code to have a pull of the same patch from someone else a few hours before mine is ready. Do you understand my logic here? It wastes the communities par effort and that's bad. Let me know who's working on what and I'll pick something that's not being developed or has been dropped and needs picking up. Happy Hump Day!

Sure, I understand that.
